src/coq/logicutils/transformation/transform.ml

```diff
@@ -89,14 +89,11 @@ let try_register_record mod_path (ind, ind') =
   try
     let r = lookup_structure ind in
     Feedback.msg_info (Pp.str "Transformed a record");
+    let con = r.s_CONST in
     let pks = r.s_PROJKIND in
-    let ps =
-      List.map
-        (Option.map (fun p -> Constant.make2 mod_path (Constant.label p)))
-        r.s_PROJ
-    in
+    let ps = r.s_PROJ in
     (try
-       declare_structure (ind', (ind', 1), pks, ps)
+       declare_structure (con, pks, ps)
```

Comment:
I don't think this is semantically equivalent, this used to be defining a new inductive type and now it's referencing only the old one